tools/generators/ethereum/contract_parsing.go

```diff
@@ -217,23 +219,63 @@ func buildMethodInfo(
 			paramDeclarations += fmt.Sprintf("%v %v,\n", paramName, goType)
 			params += fmt.Sprintf("%v,\n", paramName)
 
-			parsingFn := ""
-			switch param.Type.String() {
-			case "bytes":
-				parsingFn = "hexutil.Decode"
-			case "address":
-				parsingFn = "chainutil.AddressFromHex"
-			case "uint256":
-				parsingFn = "hexutil.DecodeBig"
-			default:
-				commandCallable = false
+			// Build CmdArgsInfo used for CLI code generator
```

Comment:
```suggestion
			// Build CmdArgsInfos used for CLI code generator
```

This is how it's named in `methodInfo` structure.

Reply:
c98f1139e3f3d2b679388ec0817916fa31e6cc5d